services/JupiterFSService.js

```diff
@@ -1,13 +1,112 @@
 import Message from '../models/message';
 import { gravity } from '../config/gravity';
+const Files = require('../models/files.js');
+const uuid = require("uuid");
+const os = require('os');
 
 const FormData = require('form-data');
 const axios = require('axios');
 
+const sendToMongo = (user, channel, fileBase64Encoded, fileName, messageObj, res, mimetype, size) => {
+  const id = uuid.v4()
+  const hostname = os.hostname();
+  const file = new Files({
+    id: id,
+    account: user.userData.account,
+    originalname: fileName,
+    mimetype: mimetype,
+    url: `https://${hostname}/v1/api/jim/file?id=${id}`,
+    size: size,
+    data: fileBase64Encoded
+  });
+
+  file.save()
+    .then((resp) => {
+      console.log(resp);
+    })
+    .catch((err) => {
+      console.log(err);
```

Comment:
if there is an error we should throw an exception here, please avoid ignoring errors